### DISCUSSION TITLE
SYS_FUNC_HALT_POW_OFF power draw when connected

### DISCUSSION THREAD

**Discussion by augmentedminds** [PiSupply/PiJuice#1032] [Q&A] [answered]
Hi all!

My current setup is as follows. I have a Raspberry Pi Zero 2W with a PiJuice Hat. The Pi Juice HAT is connected to a reasonably large external battery via the micro USB port on the Pi Juice. I am trying to build a system where the Pi Juice does the following:

 - Schedule power on/off to save external battery life. 
 - Act as a UPS if/when the external battery is drained or reaches a low power state and report back.
 
When I run the command SYS_FUNC_HALT_POW_OFF with the external battery connected via the micro USB, I still see ~250 to 300mA draw from the external battery. Are there any configuration options or solutions to only draw the 0.5 mA from the external battery like the documentation states is drawn when on Pi Juice battery alone?

I am happy to clarify any details I may have missed. I appreciate any help!

**Comment by tvoverbeek** [comment, marked as answer]
The PiJuice MCU will only go to low power mode when the 5V to the Pi and there is no external power supplied to both the RPi and the PiJuice.
Since you use an external battery as external power for the PiJuice, the PiJuice MCU will **not** switch to low power mode.
